includes/admin/class-wc-rest-payments-survey-controller.php

```diff
@@ -0,0 +1,191 @@
+<?php
+/**
+ * Class WC_REST_Payments_Survey_Controller
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for settings.
+ */
+class WC_REST_Payments_Survey_Controller extends WP_REST_Controller {
+
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc/v3';
+
+	/**
+	 * Endpoint path.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'payments/upe_survey';
```

Comment:
Do we think there is value in generalizing the endpoint for other surveys that will almost surely come up in the future("please let us know which payment method to next add", etc)? Meaning that the base would be `payments/surveys` and the REST controller would accept a survey slug in the request which is then passed along to WPCOM. We could then support adding new surveys without registering new routes. 🤔

Reply:
This survey will stay live only for a limited period of time, then its code will be removed. Maybe we can see how many new surveys we will be adding first, if any? 👀